src/libcpu/src/jit/jit_verify.h

```diff
@@ -0,0 +1,38 @@
+#include "jit_internal.h"
+#include "state.h"
+
+namespace cpu
+{
+
+namespace jit
+{
+
+struct
+VerifyBuffer
+{
+   Core coreCopy;               // Copy of core state before JIT execution
```

Comment:
Hmm.  I think the best option would be to just create a Core object at the interpreter call site and drop the CoreRegs into it, but I guess having Core could work as long as you only copy the CoreReg's part.  The point of this is that Core is meant to be non-copyable (but havn't gotten there yet).